app/models/ChannelType.scala

```diff
@@ -19,9 +19,8 @@ package models
 sealed trait ChannelType
 
 object ChannelType extends Enumerable.Implicits {
-  case object web  extends ChannelType
-  case object api  extends ChannelType
-  case object ncts extends ChannelType
+  case object web extends ChannelType
+  case object api extends ChannelType
```

Comment:
This would need to change everywhere at the moment, as its lowercase in the public API as well. I've no issue with the change, but we'd need to change it in departures, the public api and I expect the front-ends too.

Reply:
@RobertLaverick create a ticket to address this